### DISCUSSION TITLE
Ritraduzione da zero dialoghi jack-tutorial completati al 100%.

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#41] [Revisione]
dentro vi ho inserito una sorpresa con istruzioni
[jack-tutorial completato 100 x 100.zip](https://github.com/tesivo/VTMB_italian_translation_for_Unofficial_Patch/files/7836164/jack-tutorial.completato.100.x.100.zip)

**Comment by NarakuITA** [comment]
Vedo che hai fatto un buon lavoro con l'audio! Si direi che ci vuole un microfono migliore e professionale, sennò questo rovinerà l'esperienza di gioco. 
Per rispondere all'altra discussione, la tua traduzione è la più accurata, la migliore. 
Direi di concentrarsi, per adesso solo sulla traduzione, c'è molto lavoro, poi penseremo all'audio. Hai fatto un buon lavoro, ne sono sorpreso, ma purtroppo... si ci vorrebbe un attrezzatura professionale e una sala professionale.  Forse completato il lavoro, si potrebbe cercare in giro, anche dei doppiatori veri, ma dubito faccino tutto gratis, anche se fan del gioco. 

Dimenticavo, ti conviene aprire un Topic nella sessione Discussioni. Cosi su Issues ci teniamo solo i problemi riscontrati.

**Comment by crazyLillo** [comment]
Grazie!
Come ho detto era solo una prova per vedere come veniva e per dimostrare che si potrebbe fare, ma fra il dire e il fare c'è di mezzo sempre il mare.
Altro che traduzione,  sarebbe un lavoro massacrante e non remunerativo. Ecco perchè non lo si fara mai. comunque io pensavo solo a jack tutorial e basta.

Lo so che la mia traduzione è fra le più "accettabili" che VTMB abbia mai avuto finora ( si potrebbe anche migliorarla andando a ricercare termini e forme diverse ma in fondo uguali, ma a che prò?)
è un videogame, non una lettera ufficiale che devo tradurre per un ambasciatore...
Qualcuno potrebbe pensare che abbia fatto un copia e incolla con le vecchie traduzioni. bhe! mettendole a confronto vi accorgerete che la mia traduzione potrebbe sembrare simile ma è completamente diversa. 
poi è sotto gli occhi di tutti, è lì.

ahh, ho notato solo ora la sezione discussioni...si daccordo farò come dici

**Comment by tesivo** [comment]
Concordo con NarakuITA in toto.

Grande crazyLillo!

Metto le basi in discussioni per la revisione!

@crazyLillo, la tua prova è davvero buona, probabilmente, senza prenderlo come obiettivo, è possibile realizzare il doppiaggio solo di alcune parti. In ogni caso è chiaramente un extra alla traduzione che deve essere completa, e poi revisionata. Tenendo presente che c'è poi da portare la traduzione alla 10.9.


Si, le tue traduzioni/revisioni delle traduzini sono migliori, anche perchè sarebbe un controsenso se ci proponessi delle traduzioni non migliorative :D

P.S.tra l'altro esistono sintetizzatori vocali che, se ben settati, possono fare un buon lavoro per tutti quei dialoghi minori o ancora in lavorazione.

**Comment by tesivo** [comment]
Ora la sezione discussioni ha la sua categoria revisione.

**Reply by crazyLillo** [reply]
Grazie!
ho detto come la penso in un altra risposta qui in help wanted, sul doppiaggio etc..
il mio obbiettivo e arrivare anche alla 11.00, anche perchè una volta finito con la 10.7, il lavoro non sarà proprio esorbitante.
Dici che esistono sintetizzatori vocali in grado di dare intonazione etc...da usare per doppiare frasi minori?
 ho i miei dubbi, ma farò una ricerca sperando di non dover scaricare nulla. ma sono molto scettico a riguardo.
mo vedo un pò che si trova...

**Reply by tesivo** [reply]
Sì esistono, ma li userei solo se dovesse prendere piede il progetto del doppiaggio e come riempitivo delle frasi minori o in fase di doppiaggio, giusto per non avere parte del gioco doppiato e parte no.

**Reply by crazyLillo** [reply]
comunque se in seguito vorrai dedicarti al doppiaggio di qualche frase minore (intendo quelle ambientali come "crab nipple"),
con un sintetizzatore, io potrei poi prendere l'audio della frase e manipolarlo un bel po per renderlo piu credibile e accettabile.
purtroppo ho win 7, e ho visto invece che con win 10 si possono avere più pèossibilità con i sintetizzatori vocali, perchè ho visto che alcuni abbastanza sofisticati, sono solo da win 10 in poi.
